### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by hannahd**

### CODE REVIEW

**Comment by hannahd** [comment]
+1 mostly for testing

**Comment by hannahd** [comment]
-1 for testing

File: src/app/issue/issue.tpl.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <div ng-if="issueCtrl.isPullRequest()"
        class="build_header"></div>
   <div class="header">
+    <a href="{{issueCtrl.issue.html_url}}" title="Open on GitHub" class="number ng-scope"><i ng-if="issueCtrl.isPullRequest()" class="icon-code-fork"></i>{{issueCtrl.issue.number}}</a>  <span class="title ng-binding">
```
Comment (my8bird) [inline src/app/issue/issue.tpl.html:10]: If you could wrap this at some length < 100 that would help GitHub's review interface a lot

File: src/app/issue/issue.tpl.html [hunk 2]
```diff
@@ -19,39 +20,42 @@
...
-
-  <div class="milestone" ng-show="issue.milestone">
-    {{issueCtrl.issue.milestone.title}}
+  <div class="content">
+      <span class="milestone" ng-show="issue.milestone">
+        {{issueCtrl.issue.milestone.title}}
+      </span>
+      <span class="label"
+            ng-repeat="label in issueCtrl.issue.labels | nonColumnLabels"
+            style="background-color: #{{label.color}}">
```
Comment (my8bird) [inline src/app/issue/issue.tpl.html:29]: We will need to use ng-style to catch scope changes

**Pull request merged**